src/pages/index.js

```diff
@@ -291,7 +291,7 @@ class Home extends Component {
                 </CtaItem>
                 <CtaItem>
                   <ButtonLink to="/tutorial/tutorial.html" type="secondary">
-                    Haz el tutorial
+                    Toma el tutorial
```

Comment:
Same here, you mind rolling this back?
```suggestion
                    Haz el tutorial
```

Reply:
Done